proposals/lip-0045.md

```diff
@@ -129,8 +135,9 @@ We define the following constants:
 | Name                                          | Type   | Value                | Description                                                      |
 |-----------------------------------------------|--------|----------------------|------------------------------------------------------------------|
 | **Interoperability Constants**                |        |                      |                                                                  |
-| `MODULE_ID_INTEROPERABILITY`                  | uint32 | 64                   | ID of the Interoperability module.                               |
-| `MAINCHAIN_ID`                                | uint32 | 1                    | Chain ID of the Lisk mainchain.                                  |
+| `MODULE_ID_INTEROPERABILITY`                  | bytes  | TBD                  | ID of the Interoperability module.                               |
+| `OWN_CHAIN_ID`                                | bytes  | 0x00000000           | Local chain ID of the chain.                                     |
```

Comment:
As far as I see, this constant is not used for anything in the interoperability module. I also don't understand what a "local chain ID" is.

Reply:
indeed this concept is now unnecessary. It was used by the token module